### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `typos --format brief`

### CODE REVIEW

File: lib/fog/aws/requests/elasticache/create_cache_cluster.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@ class Real
         #   * :engine_version <~String> - The Cluster's caching software version
         #   * :notification_topic_arn <~String> - Amazon SNS Resource Name
         #   * :port <~Integer> - The memcached port number
-        #   * :preferred_availablility_zone <~String>
+        #   * :preferred_availabilility_zone <~String>
```
Comment (geemus) [inline lib/fog/aws/requests/elasticache/create_cache_cluster.rb:22]: This correction looks off, I think it needs one fewer `il` in the middle? That should match with the other change below and the related one in the other file above.

**Review by geemus — CHANGES_REQUESTED** [review]
Thanks for all the fixes. I think one of the corrections was inaccurate, but otherwise looking good.

**Review by geemus — APPROVED** [review]
Thanks!

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**